Source/Diagnostics/ReducedDiags/LoadBalanceCosts.cpp

```diff
@@ -49,8 +49,9 @@ void LoadBalanceCosts::ComputeDiags (int step)
     m_nBoxesMax = std::max(m_nBoxesMax, nBoxes);
 
     // resize and clear data array
-    m_data.resize(m_nDataFields*nBoxes, 0.0);
-    m_data.assign(m_nDataFields*nBoxes, 0.0);
+    size_t m_dataSize = m_nDataFields*nBoxes;
+    m_data.resize(m_dataSize, 0.0);
+    m_data.assign(m_dataSize, 0.0);
```

Comment:
```suggestion
    const size_t m_dataSize = m_nDataFields*nBoxes;
    m_data.resize(m_dataSize, 0.0);
    m_data.assign(m_dataSize, 0.0);
```
or 
```suggestion
    m_data.resize( (size_t) m_dataSize, 0.0);
    m_data.assign( (size_t) m_dataSize, 0.0);
```
(a bit old-school but compact ^^)

Reply:
Thanks, I changed as in the first suggestion.